src/modules/repository/implementation/sequelize/migrations/20240924161700-create-paranet-synced-asset.js

```diff
@@ -45,34 +45,63 @@ export const up = async ({ context: { queryInterface, Sequelize } }) => {
         },
     });
 
-    await queryInterface.sequelize.query(`
-        CREATE TRIGGER before_insert_paranet_synced_asset
-        BEFORE INSERT ON paranet_synced_asset
-        FOR EACH ROW
-        SET NEW.created_at = NOW();
+    const [triggerInsertExists] = await queryInterface.sequelize.query(`
```

Comment:
is it more efficient that we do always DROP TRIGGER IF EXISTS and then create it again? rather than running select on trigger table always